spanner/spansql/sql_test.go

```diff
@@ -232,6 +232,20 @@ func TestSQL(t *testing.T) {
 			"DELETE FROM Ta WHERE C > 2",
 			reparseDML,
 		},
+		{
+			&Update{
+				Table: "Ta",
+				Items: []UpdateItem{
+					{Column: "Cb", Value: IntegerLiteral(4)},
+					{Column: "Ce", Value: StringLiteral("wow")},
+					{Column: "Cf", Value: ID("Cg")},
+					{Column: "Ch", Value: nil},
```

Comment:
I would have expected this to set the value of `Ch` to `NULL`, regardless of the default value of `Ch`.

Reply:
I'll add a `NULL` case explicitly to make this clearer.